### PR TITLE
HZN-1290: add Sentinel to the system test containers

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds the Sentinel container as an option for smoke tests, as well as to the `.all()` method when creating test environments.

### CODE REVIEW

File: docker/sentinel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Ports
+# 162/1162 - SNMP traps
+# 1399     - RMI Registry
+# 514/1514 - Syslog
+# 8301     - Karaf SSH
+# 46444    - RMI Server
+# 50000    - Telemetry
+# 50001    - Telemetry
+# 50002    - Telemetry
+EXPOSE 162/udp 1162/udp 1399 514/udp 1514/udp 5150 8301 46444 50000/udp 50001/udp 50002/udp 50003/udp
```
Comment (j-white) [inline docker/sentinel/Dockerfile]: We don't need to expose the SNMP/Syslog or Telemetry ports here.

File: docker/sentinel/etc/org.opennms.netmgt.syslog.cfg [hunk 2]
```diff
@@ -0,0 +1 @@
+syslog.listen.interface = 0.0.0.0
```
Comment (j-white) [inline docker/sentinel/etc/org.opennms.netmgt.syslog.cfg]: Don't need this.

File: docker/sentinel/etc/org.opennms.netmgt.trapd.cfg [hunk 3]
```diff
@@ -0,0 +1 @@
+trapd.listen.interface = 0.0.0.0
```
Comment (j-white) [inline docker/sentinel/etc/org.opennms.netmgt.trapd.cfg]: Don't need this either.

File: src/main/java/org/opennms/test/system/api/NewTestEnvironment.java [hunk 4]
```diff
@@ -595,6 +614,65 @@ private void spawnSnmpd() throws DockerException, InterruptedException, IOExcept
...
+                }
+            });
+        }
+
+        final List<String> binds = new ArrayList<>();
+        binds.add(sentinelOverlay.toString() + ":/sentinel-docker-overlay");
+        binds.add(sentinelKarafLogs.toString() + ":/opt/sentinel/data/log");
+
+        final List<String> links = Lists.newArrayList();
+        //links.add(String.format("%s:opennms", containerInfoByAlias.get(ContainerAlias.OPENNMS).name()));
```
Comment (j-white) [inline src/main/java/org/opennms/test/system/api/NewTestEnvironment.java]: Can we link to `opennms`, `elasticsearch` and `postgres` here? Don't need to link to `snmpd` or `tomcat`.

**Review by j-white — COMMENTED** [review]
👍

**Comment by mvrueden** [comment]
LGTM.

**Pull request merged**